helm/templates/deployment.yaml

```diff
@@ -92,6 +92,8 @@ spec:
               value: {{ $gpkgPath }}
             - name: GENERAL__batchSize
               value: {{ .Values.env.general.batchSize | quote }}
+            - name: GENERAL__batchMaxBytes
+              value: {{ .Values.env.general.batchMaxBytes | quote }}
```

Comment:
I also added this option to the helm chart (which I missed before)
not sure how your cicd flow works exactly and if its needed here so if not, let me know

Reply:
Good thing you noticed! Thank you very much.
Sadly our CI/CD doesn't add this automatically.